keda/templates/manager/deployment.yaml

```diff
@@ -191,8 +191,6 @@ spec:
           resources:
             {{- if .Values.resources.operator }}
             {{- toYaml .Values.resources.operator | nindent 12 }}
-            {{- else }}
-            {{- toYaml .Values.resources | nindent 12 }}
```

Comment:
I'm not sure about removing this fallback behavior, although I see the point of the PR, removing this can be disuptive

Reply:
I could leave the fallback but it would fallback to the empty object which seems to be pointless? IMO resources/limits should be set explicitly.

Reply:
My afraid is about doing breaking changes if someone is relying on default values. @kedacore/keda-helm-contributors

Reply:
I don't think it's a good idea to remove the default values, this might be a breaking change for some users.